cmd/zc_traverser_file.go

```diff
@@ -194,14 +201,24 @@ func (t *fileTraverser) Traverse(preprocessor objectMorpher, processor objectPro
 				return fmt.Errorf("cannot list files due to reason %s", err)
 			}
 			for _, fileInfo := range lResp.FileItems {
+				// These conditions are to prevent a GetProperties from returning a 404 when trailing dot is turned off.
```

Comment:
Ditto

Reply:
this is just a log for warning, so when customers experience weird behavior, they can look at the logs and see why